packages/polling-controller/src/PollingController.test.ts

```diff
@@ -211,6 +211,40 @@ describe('PollingController', () => {
       await Promise.resolve();
       expect(controller.executePoll).toHaveBeenCalledTimes(2);
     });
+    it('should start and stop poll for the same networkClientId and different options', async () => {
```

Comment:
[nit] can we add one more test `'should start and stop polling sessions for different networkClientIds with the same options'